modules/claims_api/app/services/claims_api/disability_compensation/pdf_generation_service.rb

```diff
@@ -5,26 +5,27 @@
 
 module ClaimsApi
   module DisabilityCompensation
-    class PdfGenerationService < ClaimsApi::Service
+    class PdfGenerationService < ServiceBase
       EVSS_DOCUMENT_TYPE = 'L023'
+      LOG_TAG = '526_v2_PDF_Generator_job'
```

Comment:
I like the discussion back and forth on better defining the tag, but one caveat is that these particular calls are no longer sidekiq jobs so not sure if `_job` really defines it well anymore. Not a huge concern but it could help distinguish the async jobs vs the synchronous calls. Same for the `DockerContainerService`

Reply:
Good call; updated log tags to use "_service" instead of "_job".